specification/recoveryservices/resource-manager/Microsoft.RecoveryServices/stable/2021-04-01/vaults.json

```diff
@@ -0,0 +1,1803 @@
+              "description": "End Time of the Resource Move Operation",
+              "type": "string",
+              "format": "date-time",
+              "readOnly": true
+            },
+            "sourceResourceId": {
+              "description": "Source Resource of the Resource Move Operation",
+              "type": "string",
+              "readOnly": true
+            },
+            "targetResourceId": {
+              "description": "Target Resource of the Resource Move Operation",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        },
+        "moveState": {
+          "description": "The State of the Resource after the move operation",
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "Unknown",
+            "InProgress",
+            "PrepareFailed",
+            "CommitFailed",
+            "PrepareTimedout",
+            "CommitTimedout",
+            "MoveSucceeded",
+            "Failure",
+            "CriticalFailure",
+            "PartialSuccess"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceMoveState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "IdentityData": {
+      "required": [
+        "type"
+      ],
+      "type": "object",
+      "description": "Identity for the resource.",
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of managed identity used. The type 'SystemAssigned, UserAssigned' includes both an implicitly created identity and a set of user-assigned identities. The type 'None' will remove any identities.",
+          "enum": [
+            "SystemAssigned",
+            "None",
+            "UserAssigned",
+            "SystemAssigned, UserAssigned"
+          ],
+          "x-ms-enum": {
+            "name": "ResourceIdentityType",
+            "modelAsString": true
+          }
+        },
+        "userAssignedIdentities": {
+          "description": "The list of user-assigned identities associated with the resource. The user-assigned identity dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}'.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/UserIdentity"
+          }
+        }
+      }
+    },
+    "UserIdentity": {
+      "type": "object",
+      "description": "A resource identity that is managed by the user of the service.",
+      "properties": {
+        "objectId": {
```

Comment:
>objectId [](http://example.com/codeflow?start=9&length=8)

Make sure to use principalId based on our [docs](https://armwiki.azurewebsites.net/authorization/managed_identities/MSIOnboardingUserAssigned.html#resource-definition-response)

Reply:
Updated